packages/types/config/index.d.ts

```diff
@@ -25,7 +25,7 @@ export { ServerMiddleware } from './server-middleware'
 /**
  * @deprecated Use NuxtConfig instead
 */
-export interface Configuration extends Record<string, any> {}
+export type Configuration = Record<string, any>
```

Comment:
Using interface was intentional i guess to be able extending. /cc @kevinmarrec

Reply:
Makes sense! Reveting back here 😋